src/Microsoft.AspNetCore.Dispatcher/SimpleEndpointHandlerFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Dispatcher
+{
+    public class SimpleEndpointHandlerFactory : HandlerFactoryBase
+    {
+        public SimpleEndpointHandlerFactory(HandlerFactoryBaseServices services) 
+            : base(services)
+        {
+        }
```

Comment:
One nice thing about this is that VS will generate this constructor for you verbatim